src/frontend/sharedComponents/ThumbnailScrollView.tsx

```diff
@@ -32,7 +32,11 @@ type ThumbnailProps = {
 export const Thumbnail = ({photo, style, size, onPress}: ThumbnailProps) => {
   const [error, setError] = React.useState(false);
 
-  const uri = 'thumbnailUri' in photo ? photo.thumbnailUri : undefined;
+  const uri =
+    (photo && 'thumbnailUri' in photo && photo.thumbnailUri) || undefined;
+  const isCapturing =
+    (photo && 'capturing' in photo && (photo.capturing as boolean)) ||
+    undefined;
```

Comment:
Nitpick: I would expect `isCapturing` to be a boolean, not `undefined | boolean`. Can you make sure it's always a bool?

Reply:
Yeah good point 👍